test/Nancy.Tests/Unit/ResponseExtensionsFixture.cs

```diff
@@ -177,5 +174,23 @@ public void Should_set_status_code_int()
 
             respone.StatusCode.ShouldEqual(HttpStatusCode.NotFound);
         }
+
+        private string GetLocation()
+        {
+#if DNX
+            return Microsoft.Extensions.PlatformAbstractions.PlatformServices.Default.Application.ApplicationBasePath;
+#else
+            return Path.GetDirectoryName(this.GetType().Assembly.Location);
+#endif
+        }
+
+        private string GetFilePath()
+        {
+#if DNX
+            return Path.Combine("Resources", "test.txt");
+#else
+            return Path.GetFileName(this.GetType().Assembly.Location);

```

Comment:
We can't check the same file in `DNX`. On `DNX` we look for a file that's in `Nancy.Tests/Resources` and for non-`DNX` we just use the assembly name `Nancy.Tests.dll` because we end up in a temp folder